src/test/java/testCases/Validate_SelfExlcusion_Limits.java

```diff
@@ -0,0 +1,171 @@
+package testCases;
+
+import java.util.concurrent.TimeUnit;
+import java.util.Random;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.SkipException;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+
+import appModules.Login_Action;
+import io.github.bonigarcia.wdm.ChromeDriverManager;
+import pageObjects.Limits;
+import pageObjects.MainAccount_Menu;
+import pageObjects.PersonalDetails_Page;
+import pageObjects.Profile_Page;
+import pageObjects.Security_Page;
+import pageObjects.Self_Exclusion;
+import utility.Constant;
+
+public class Validate_SelfExlcusion_Limits {
+	public WebDriver driver;
+	int getAccBal , getMaxPos;
+	//Test Method to login into the site
+	@Test(priority=0)
+	public void Login() {
+	  	Login_Action.Execute(driver,Constant.Email,Constant.Password);
+	}
+	//Test Method to navigate to self exclusion page
+	@Test(priority=1)
+	public void NavigateToPersonalDetailsPage() {
+		  	MainAccount_Menu.link_MainAccount(driver).click();
+			MainAccount_Menu.link_Security(driver).click();
+			Security_Page.link_SelfExclusion(driver).click();
+	}
+	//Test Method to test error message when the new amount exceed the current amount
+	@Test(priority=2)
+	public void Test_Exceed_MaxAccountBalance() {
+		//If there is no value added, skip this test		
+		if(Self_Exclusion.txt_MaxAccCashBal(driver).getAttribute("value").equals("")){
+			throw new SkipException("Skipping this exception");}
+		else{
+			//Increase the current value
+			int temp;
+			temp = Integer.parseInt(Self_Exclusion.txt_MaxAccCashBal(driver).getAttribute("value"));
+			getAccBal = temp + 1;
+			Self_Exclusion.txt_MaxAccCashBal(driver).clear();
+			Self_Exclusion.txt_MaxAccCashBal(driver).sendKeys(Integer.toString(getAccBal));
+			//If the error message is shown
+			if(Self_Exclusion.err_MaxAccBal(driver).isDisplayed()){
+				System.out.println("[P]New amount are not allowed to exceed current amount.");;
```

Comment:
Can we put assertEqual() here ? and just getText() for the error message - we can validate the correct error message